processor/transformer/transformer.go

```diff
@@ -86,35 +94,48 @@ type HandleT struct {
 	receivedStat stats.Measurement
```

Comment:
this is a good chance to rename some structs:
- `HandleT` --> `Handle`
- `TransformerEvent` --> `TransformerEvent`
- `MetadataT` --> `Metadata`
- `ResponseT` --> `Response`

Reply:
I thought of doing it but it was touching too many files so I skipped.

Reply:
The refactoring tools available in IDEs are powerful enough and can help performing such operations in a safe manner, no matter the amount of files being modified.